snuba/util.py

```diff
@@ -155,6 +155,23 @@ def is_function(column_expr: Any, depth: int=0) -> Optional[Tuple[Any, ...]]:
         return None
 
 
+def is_alias_column_expr(column_expr: Any) -> bool:
+    """
+    Returns True if column_expr is an alias column expression, otherwise False.
+
+    An alias column expression is a 3-tuple of (column_name, None, alias)
+    """
+    if (
```

Comment:
nit: you can turn this into a single return statement and remove the id expression:
```
return isinstance(.... .and ......
```